stackinator/builder.py

```diff
@@ -432,13 +432,14 @@ def generate(self, recipe):
                 """\
 repo:
   namespace: alps
+  api: v2.0
 """
             )
 
         # create the repository step 2: create the repos.yaml file in build_path/config
         repos_yaml_template = jinja_env.get_template("repos.yaml")
         with (config_path / "repos.yaml").open("w") as f:
-            repo_path = recipe.mount / "repo"
+            repo_path = recipe.mount / pathlib.Path("spack_repo", "alps")
```

Comment:
Sorry, might as well change both if we're changing one:
```suggestion
            repo_path = recipe.mount / "spack_repo" / "alps"
```